source/projects/private-voice-assistant/voice_remote_classic_phone.markdown

```diff
@@ -0,0 +1,154 @@
+---
+title: "Setting up an analog phone to control Home Assistant"
+---
```

Comment:
Rename this file to `index.markdown` so the url becomes `/projects/private-voice-assistant`